mono/mini/method-to-ir.c

```diff
@@ -7005,8 +7006,23 @@ is_supported_tail_call (MonoCompile *cfg, MonoMethod *method, MonoMethod *cmetho
 		supported_tail_call = FALSE;
 	if (cmethod->wrapper_type && cmethod->wrapper_type != MONO_WRAPPER_DYNAMIC_METHOD)
 		supported_tail_call = FALSE;
-	if (call_opcode != CEE_CALL)
+
+	g_assert (call_opcode == CEE_CALL || call_opcode == CEE_CALLVIRT || call_opcode == CEE_CALLI);
+	switch (call_opcode)
+	{
```

Comment:
This should be on the previous line.,

Reply:
This comment I don't understand.

Reply:
Its the coding convention, { should be on the same line as the switch.

Reply:
Oh, the brace, right.